browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -193,6 +193,7 @@ WebUIFactoryFunction GetWebUIFactoryFunction(WebUI* web_ui,
       ((url.host_piece() == kWelcomeHost ||
         url.host_piece() == chrome::kChromeUIWelcomeURL) &&
        !profile->IsGuestSession()) ||
+      url.host_piece() == kBraveGettingStartedHost ||
```

Comment:
Do you need this in here? This PR is using `ChromeWebUIConfigs`...

Reply:
This was a leftover - thanks!